prime-router/src/main/kotlin/history/azure/DeliveryFunction.kt

```diff
@@ -45,9 +47,13 @@ class DeliveryFunction(
      * @return Name for the organization
      */
     override fun getOrgName(organization: String): String? {
-        val receiver = workflowEngine.settings.findReceiver(organization)
-        receivingOrgSvc = receiver?.name
-        return receiver?.organizationName
+        return if (organization.contains(Sender.fullNameSeparator)) {
```

Comment:
They're literally the same character so no logic would change, but shouldn't this be `Receiver.fullNameSeparator` given the context?

Reply:
I agree, but in `Receiver.kt:parseFullName()`, `Sender.fullNameSeparator` is also used. Intentionally or otherwise, I'm unsure.

I'm happy to change it in both places. However, I have a conceptualized refactor that would render that moot. More on that in a later reply.